src/command/pss/pss-command.ts

```diff
@@ -1,23 +1,17 @@
 import { Option } from 'furious-commander'
-import { getTopic } from '../../utils'
+import { topicProperties, topicStringProperties } from '../../utils/option'
 import { RootCommand } from '../root-command'
 
 export class PssCommand extends RootCommand {
-  @Option({
-    key: 'topic',
-    alias: 't',
-    description: 'PSS topic',
-    default: '0'.repeat(64),
-    defaultDescription: '32 zero bytes',
-  })
+  @Option(topicProperties)
   public topic!: string
 
-  @Option({ key: 'hash-topic', alias: 'H', type: 'boolean', description: 'Hash the topic to 32 bytes', default: false })
-  public hashTopic!: boolean
+  @Option(topicStringProperties)
+  public topicString!: string
 
   public init(): void {
     super.init()
 
-    this.topic = getTopic(this.bee, this.console, this.topic, this.hashTopic)
+    this.topic = this.topic || this.bee.makeFeedTopic(this.topicString)
```

Comment:
So as it was discussed under [a bee-js PR](https://github.com/ethersphere/bee-js/pull/343#discussion_r648490565), there is no way currently to pass hashed topic from user side, so this part is wrong, but could be fixed later in an other PR with resolving [this](https://github.com/ethersphere/swarm-cli/issues/141) issue